src/main/java/com/acoustic/salarycalculator/menu/SalaryCalculatorMenu.java

```diff
@@ -1,27 +1,27 @@
-package org.example.enumApproch.menu;
+package com.acoustic.salarycalculator.menu;
 
 import java.math.BigDecimal;
 import java.util.Arrays;
 import java.util.Scanner;
 
-import org.example.enumApproch.enums.JobsCategory;
-import org.example.enumApproch.print.PrinterSalaryCalculator;
+import com.acoustic.salarycalculator.jobscategories.JobsCategory;
+import com.acoustic.salarycalculator.printer.SalaryCalculatorPrinter;
 
 
-public class MenuSalaryCalculator {
+public class SalaryCalculatorMenu {
 
     private final Scanner scanner;
-    private final PrinterSalaryCalculator printerCalculator;
+    private final SalaryCalculatorPrinter printerCalculator;
 
-    public MenuSalaryCalculator() {
+    public SalaryCalculatorMenu() {
         this.scanner = new Scanner(System.in);
-        this.printerCalculator = new PrinterSalaryCalculator();
+        this.printerCalculator = new SalaryCalculatorPrinter();
 
```

Comment:
in the metod surveyINput you could just ask if someone want to take participate in survey, if yes (or y) then go ahead, all other cases just finish program, don't need to put no, or checking for other types , so you can remove loop and try catch etc..

Reply:
fixed